source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/input/customcontrols/ValidatedCheckBox.java

```diff
@@ -0,0 +1,72 @@
+package io.adaptivecards.renderer.input.customcontrols;
+
+import android.content.Context;
+import android.graphics.Color;
+import android.graphics.PorterDuff;
+import android.support.v4.content.ContextCompat;
+import android.widget.CheckBox;
+import android.widget.LinearLayout;
+
+import io.adaptivecards.R;
+
+public class ValidatedCheckBox extends CheckBox implements IValidatedInputView
+{
+
+    public ValidatedCheckBox(Context context)
```

Comment:
>ValidatedCheckBox [](start = 11, length = 17)

how come we are selective about where we are adding documentation headers in this file? whats your criteria?

Reply:
Added documentation headers for all new elements